3336/0/triangle.rb

```diff
@@ -0,0 +1,38 @@
+# rubocop:disable all
+# frozen_string_literal:true
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+# :reek:FeatureEnvy
+def triangle(a_side, b_side, c_side)
+  raise TriangleError if (a_side <= 0) || (b_side <= 0) || (c_side <= 0)
+  raise TriangleError if (a_side + b_side <= c_side) || (b_side + c_side <= a_side) || (c_side + a_side <= b_side)
+
+  if (a_side == b_side) & (b_side == c_side)
+    :equilateral
+  elsif (a_side == b_side) || (b_side == c_side) || (a_side == c_side)
+    :isosceles
+  else
+    :scalene
+  end
+end
+
+# Error class used in part 2.  No need to change this code.
+class TriangleError < StandardError
+end
+# comment for hound check commit
+# rubocop:enable all
```

Comment:
this line looks suspicious. I'd say that this symbol at the end of line shouldn't be here.

Reply:
@anatoliliotych  
+ Link to the video
+ Used commit guide on new commits 
+ Unnecessary comments deleted.